style/game.css

```diff
@@ -130,33 +140,3 @@ header h1 span.gray {
   color: var(--white);
   background-color: var(--dark-gray);
 }
-
-/* Responsive min-width: 425px */
-
-@media (min-width: 425px) {
-  /* Responsive header */
-
-  header {
-    padding: 5px 0;
-  }
-
-  /* Responsive board */
-
-  .board {
-    max-width: calc(var(--box-size-larger) * 5 + 20px);
-    padding: 1.5rem 0;
-  }
-
-  /* Responsive guess */
-
-  .guess {
-    margin-bottom: 5px;
-  }
-
-  /* Responsive box */
-
-  .box {
-    height: var(--box-size-larger);
-    width: var(--box-size-larger);
-  }
-}
```

Comment:
Removing these responsive rules as they were conflicting with the new virtual keyboard.